app/models/long_submission.rb

```diff
@@ -40,11 +40,23 @@ class LongSubmission < ActiveRecord::Base
 
   validates :score,
             numericality: { allow_nil: true, greater_than_or_equal_to: 0 }
+  def get_score_text
+    #scorehash = [*0..LongProblem.maximum("max_score")].each_with_object({}) do |item, memo|
+    #  memo[item] = item.to_s
+    #end
+    #scorehash[nil] = '-'
+    #return scorehash
+    return "-" if score.nil?
+    return score.to_s
```

Comment:
You can omit `return` on the last statement on Ruby functions. Ruby functions always return the value of the last statement in the function body.